### ISSUE TITLE
Forgot tensorflow import

### ISSUE THREAD

**Issue by yassineAlouini** [mkocabas/focal-loss-keras#15]
In the https://github.com/mkocabas/focal-loss-keras/blob/master/focal_loss.py file, you forgot to import 
tensorflow: `import tensorflow as tf`.

**Comment by chengdabao** [comment]
u r write

**Comment by chengdabao** [comment]
u r right

**Comment by mkocabas** [comment]
https://github.com/mkocabas/focal-loss-keras/commit/c40cacff60bda15b02bd83ea3a1b2be739d56117 should solve the issue.
